index.js

```diff
@@ -65,18 +65,45 @@ menuLinkContactElement.addEventListener('click', () => {
 
 // ----------------------- contact validation ---------------------------------
 const formElement = document.getElementById('contact-form');
-// const formNameElement = document.getElementById('name');
+const formNameElement = document.getElementById('name');
 const formEmailElement = document.getElementById('email');
 
 formElement.addEventListener('submit', (event) => {
   const email = formEmailElement.value;
   const errmessages = [];
   if (email.toLowerCase() !== email) {
     event.preventDefault();
-    errmessages.push('email must all be in lowercase letters. form was not submitted');
+    errmessages.push(
+      'email must all be in lowercase letters. form was not submitted',
+    );
     const emailErrElement = document.getElementById('email-err');
-    // Destructure the entire `errmessages` array and access the first element using an index
     const [firstErrorMessage] = errmessages;
     emailErrElement.innerText = firstErrorMessage;
   }
 });
+
+const formData = {
+  commentElement: document.getElementById('comment-box'),
+};
+
+formNameElement.onchange = () => {
+  formData.nameElement = document.getElementById('name').value;
+  localStorage.setItem('nameInput', formData.nameElement);
+};
+
+formEmailElement.onchange = () => {
+  formData.emailElement = document.getElementById('email').value;
+  localStorage.setItem('emailInput', formData.emailElement);
```

Comment:
- [ ] You should use `JSON.stringify` to store the data in the local storage as a string kindly implement this.
- [ ] Also the data from the local storage are not added to the form only after the user clicks the inputs, kindly make sure the form is populated after the page reloads directly